src/GitTfs/GitTfs.cs

```diff
@@ -63,6 +74,47 @@ private void UpdateLoggerOnDebugging()
                 //consoleRule.DisableLoggingForLevel(LogLevel.Trace);
                 LogManager.ReconfigExistingLoggers();
             }
+
+            if (!String.IsNullOrEmpty(_globals.LogFilePath))
+            { 
+                Trace.WriteLine("Creating new log file [" + _globals.LogFilePath + "]");
+
+                try
+                {
+                    //Step 1.Create configuration object
+//                    var config = LogManager.Configuration;
```

Comment:
remove? (and update the `Step n.` comments)

Reply:
removed